src/visualization.cpp

```diff
@@ -75,34 +75,66 @@ void visualize(const World& world, const std::vector<Robot*>& robots, bool colli
         const Object& robot = world.object((*it)->robot_id);
         cv::Scalar robot_color(0, 0, 255);
 
-        std::vector<geo::Vector3> robot_points;
-        robot_points.push_back(geo::Vector3( 0.1,  -0.2, 0));
-        robot_points.push_back(geo::Vector3( 0.1,  -0.1, 0));
-        robot_points.push_back(geo::Vector3( 0.05, -0.1, 0));
-        robot_points.push_back(geo::Vector3( 0.05,  0.1, 0));
-        robot_points.push_back(geo::Vector3( 0.1,   0.1, 0));
-        robot_points.push_back(geo::Vector3( 0.1,   0.2, 0));
-        robot_points.push_back(geo::Vector3(-0.1,   0.2, 0));
-        robot_points.push_back(geo::Vector3(-0.1,  -0.2, 0));
+        std::vector<geo::Vector3> robot_head_points;
+        std::vector<geo::Vector3> eye_points;
+        std::vector<geo::Vector3> eye_pupil_points;
+        std::vector<geo::Vector3> lrf_point;
+
+        robot_head_points.push_back(geo::Vector3( 0.075, -0.175, 0));
+        robot_head_points.push_back(geo::Vector3( 0.075,  0.175, 0));
+        robot_head_points.push_back(geo::Vector3(-0.075,  0.175, 0));
+        robot_head_points.push_back(geo::Vector3(-0.075, -0.175, 0));
+        eye_points.push_back(geo::Vector3( 0.0,  0.075, 0));
+        eye_points.push_back(geo::Vector3( 0.0, -0.075, 0));
+        eye_pupil_points.push_back(geo::Vector3( 0.015,  0.075, 0));
+        eye_pupil_points.push_back(geo::Vector3( 0.015, -0.075, 0));
+        lrf_point.push_back(geo::Vector3( 0.175, 0.0, 0));
```

Comment:
Misschien alle afmetingen schalen met de robot radius.

Reply:
Mogelijk een idee idd, dan is wel de vraag wat is het nut, als de viz toch aangepast moet worden als de robot veranderd